pkg/clients/postgres/clients.go

```diff
@@ -482,7 +482,7 @@ func PageQuery(pm clients.Page) (string, error) {
 	if pm.Name != "" {
 		query = append(query, "name ILIKE '%' || :name || '%'")
 	}
-	if pm.Identity != "" {
+	if pm.Identity != "" && pm.Role == clients.AdminRole {
```

Comment:
Instead of chaning here in service layer, If  request user is not user admin , then create new pagemeta with limited (required) fields like only with id, name or may be you can add  tag

Reply:
I have implemented as per your suggestion but the superadmin role is not set to `all` anywhere so I have left that line of code.